apps/admin/frontend/src/screens/election_screen.tsx

```diff
@@ -46,14 +53,27 @@ export function DefinitionScreen(): JSX.Element {
               {format.localeDate(new Date(election.date))}
             </P>
           </div>
-        </ElectionMetadataContainer>
-        <Button
-          variant="danger"
-          icon="Delete"
-          onPress={() => setIsRemovingElection(true)}
-        >
-          Remove Election
-        </Button>
+        </ElectionCard>
+        {isSystemAdministratorAuth(auth) && (
+          <Button
+            color="danger"
+            icon="Delete"
+            onPress={() => setIsRemovingElection(true)}
+          >
+            Remove Election
+          </Button>
+        )}
+        {isElectionManagerAuth(auth) && (
+          <React.Fragment>
+            <P>
+              Save the Ballot Package to USB to configure VxCentralScan or
+              VxScan.
+            </P>
+            <P>
+              <ExportElectionBallotPackageModalButton />
+            </P>
+          </React.Fragment>
+        )}
```

Comment:
In development, I've always wanted to export the ballot package as the system admin 🤷 , but making a decision on that doesn't need to happen in this PR / code freeze.

Reply:
Oh interesting. I agree we can discuss outside of this PR